### PR TITLE
add legal files to built artifacts

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

**Review by m0mus — APPROVED** [review]
LGTM

File: nucleus/parent/pom.xml [hunk 1]
```diff
@@ -70,6 +70,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <legal.doc.source>${maven.multiModuleProjectDirectory}</legal.doc.source>
```
Comment (romain-grecourt) [inline nucleus/parent/pom.xml]: Are there any requirement for this property to work ?

Comment (lukasj) [reply nucleus/parent/pom.xml]: correct env setup... given that there exists https://issues.apache.org/jira/browse/MNG-5830.
I think using rootlocation looks better as it is not tight to from where or where the build started

Comment (lukasj) [reply nucleus/parent/pom.xml]: the only drawback is that it runs for every module. Or I can move it to the root pom if you prefer that

Comment (romain-grecourt) [reply nucleus/parent/pom.xml]: This needs to run for every module, correct ? The updated version looks fine IMO.

**Review by romain-grecourt — APPROVED** [review]

File: nucleus/parent/pom.xml (hunk 1 above)
Comment (lukasj) [reply nucleus/parent/pom.xml]: yes, every produced source and binary jar should contain these file

**Pull request merged**